app/src/main/java/org/greenstand/android/TreeTracker/models/Tree.kt

```diff
@@ -31,5 +30,6 @@ class Tree(
         // a tree minus the last absolute step count recorded when capturing a previous tree. This
         // is the indicator for the number of steps taken between two trees.
         const val DELTA_STEP_COUNT_KEY = "delta_step_count"
+        const val DBH_ATTR_KEY = "dbh"

```

Comment:
Since DBH is a standard abbreviation (see wiki link above) I'm thinking it may make sense to leave it as DBH. I'd like searches for DBH to show all it's usages in the code.

Reply:
I still think a one line comment would help, eliminates the need to do a search lookup.